website/project/metadata/utils.py

```diff
@@ -0,0 +1,33 @@
+def serialize_meta_schema(meta_schema):
+    if not meta_schema:
+        return None
```

Comment:
Here is an emoticon of an octopus, *for absolutely no reason whatsoever*. :palm_tree:  :octopus: :palm_tree: 

10 hrs, and around halfway through.